src/check_mysql_conf.py

```diff
@@ -51,6 +51,11 @@ def parse_args():
         '--passwd',
         help='MySQL password (default: %(default)s)'
     )
+    parser.add_argument(
+        '--ignore_vars',
+        default='wsrep_sst_auth',
+        help='Varaibles that don\'t get checked (default: %(default)s)'
```

Comment:
```suggestion
        help="Variables that don't get checked (default: %(default)s)"
```
Nice with the `%(default)`, I didn't even know that. 👍🏼